gen/CCompile_gen.ml

```diff
@@ -1139,9 +1139,10 @@ module Make(O:Config) : Builder.S
   else dump_c_test
  *)
 
-      let test_of_cycle name ?com ?(info=[]) ?(check=(fun _ -> true)) ?scope ?(init=[])
+      let test_of_cycle name ?com ?(info=[]) ?(check=(fun _ -> true)) ?scope ?(init=[]) ?(init_pte=[])
           es c =
         ignore (scope) ;
+        ignore (init_pte) ;
```

Comment:
Can you help me understand this? Aren't you passing `init_pte` as an argument which is then unused?